cymetric/metrics.py

```diff
@@ -285,12 +285,11 @@ def agents(entry, exit, decom, info):
     idx = ['SimId','AgentId','ExitTime']
     df = entry[['SimId', 'AgentId', 'Kind', 'Spec', 'Prototype', 'ParentId',
                 'Lifetime', 'EnterTime']]
-    df['ExitTime'] = [np.nan]*len(entry)
-    print(df)
+    df['Exits'] = [np.nan]*len(entry)
     if exit is not None:
         df = tools.merge_and_fillna_col(df, exit[['SimId','AgentId','ExitTime']], 
-                                        'ExitTime','ExitTime',on=mergeon)
-    print(df)
+                                        'Exits','ExitTime',how='left',on=mergeon)
```

Comment:
```suggestion
                                        'Exits','ExitTime', on=mergeon)
```
this is probably not required.